src/react/databrowser/buckets/details/Overview.tsx

```diff
@@ -56,26 +59,22 @@ function getDefaultBucketRetention(bucketInfo: BucketInfo): string {
   return 'Inactive';
 }
 
-function canDeleteBucket(
-  bucketName: string,
-  loading: boolean,
-  replicationStreams: Array<Replication>,
-) {
+function canDeleteBucket(loading: boolean) {
   if (loading) {
     return false;
   }
 
-  const checkBucketStream = replicationStreams.find((r) => {
-    return r.source.bucketName === bucketName;
-  });
-
-  if (checkBucketStream) {
-    return false;
-  }
-
   return true;
 }
 
+const StyledLink = styled(Link)`
+  color: ${(props) => props.theme.brand.textLink};
+  text-decoration: none;
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
```

Comment:
Yes, I copied the styles from CellLink, I'll reuse it but we have to keep in mind that if we update it in a way which makes the styling more appropriate for table elements, it might not look good in the modal. There's NameLink which is similar but it has padding right and it's styled div and yeah, using divs for links is not ideal

Reply:
` divs for links is not ideal`: worth than that it is not accessible !